stdlib/socket/src/main/ballerina/src/socket/Module.md

```diff
@@ -1,154 +1,28 @@
-## Module overview
+This module provides an implementation for connecting to a remote socket server or acts as a server for an incoming socket request. The module facilitates three types of endpoints called `socket:Client`, `socket:UdpClient`, and `socket:Listener`.
```

Comment:
"provides an implementation for connecting to a remote socket server or acts as a server for an incoming socket request" 
^ The above doesn't sound right. Need to be rephrased properly.

Also, the "endpoint" terminology was dropped some time back. Again, needs to be rephrased in a better way